### PR TITLE
Add prow-addons-ctrl job and slack token secret information and checks

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Changes proposed in this pull request:

- Add prow-addons-ctrl job and slack token secret information and checks

**Related issue(s)**

-  #466

### CODE REVIEW

File: docs/prow/presets.md [hunk 1]
```diff
@@ -25,3 +25,4 @@ This document contains the list of all Presets available in the [`config.yaml`](
 | **preset-bot-github-ssh** | It connects the ssh key of the bot account to your job and sets the value with the path to this key.                    |
 | **preset-kyma-artifacts-bucket** | It defines the environment variable for the Kyma's artifact bucket.                     |
 | **preset-stability-checker-slack-notifications** | It defines a webhook URL and a client token required for the Slack integration.                 |
+| **preset-slack-internal-bot-token** | It sets the environment variable with the Slack token for the bot account in internal workspace. More information can bo found [here](https://api.slack.com/docs/token-types#bot).|
```
Comment (aszecowka) [inline docs/prow/presets.md]: preset-slack-token-for-prowjob-failures-notification ?

File: prow/jobs/test-infra/prow-addons-ctrl-manager.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        requests:
+          memory: 1.5Gi
+          cpu: 0.8
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository-test-infra: "true"
+
+presubmits: # runs on PRs
```
Comment (aszecowka) [inline prow/jobs/test-infra/prow-addons-ctrl-manager.yaml]: where can I find tests for this yaml file :) ?

Comment (mszostok) [reply prow/jobs/test-infra/prow-addons-ctrl-manager.yaml]: 🤦‍♂️

File: docs/prow/presets.md (hunk 1 above)
Comment (mszostok) [reply docs/prow/presets.md]: discussed

File: docs/prow/prow-installation-on-forks.md [hunk 3]
```diff
@@ -78,6 +78,7 @@ For the purpose of the installation, you must have a set of service accounts and
  - **sa-gcr-push-kyma-project** with the role that allows the account to push images to Google Container Repository in the `kyma-project` GCP project:
     - Storage Admin (`roles/storage.admin`)
  - **kyma-bot-npm-token** which is a token for publishing npm packages
+ - **sap-slack-bot-token** which is a token for publishing messages to SAP CX workspace. More information can bo found [here](https://api.slack.com/docs/token-types#bot).
```
Comment (klaudiagrz) [inline docs/prow/prow-installation-on-forks.md]:
```suggestion
 - **sap-slack-bot-token** which is a token for publishing messages to the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot).
```

File: development/tools/jobs/testinfra/prow_addons_ctrl_manager_test.go [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	"testing"
+
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const prowAddonsCtrlManagerJobPath = "./../../../../prow/jobs/test-infra/prow-addons-ctrl-manager.yaml"
+
+func TestBucJobsPresubmit(t *testing.T) {
```
Comment (aszecowka) [inline development/tools/jobs/testinfra/prow_addons_ctrl_manager_test.go]: c`mon

File: docs/prow/presets.md [hunk 5]
```diff
@@ -25,3 +25,4 @@ This document contains the list of all Presets available in the [`config.yaml`](
 | **preset-bot-github-ssh** | It connects the ssh key of the bot account to your job and sets the value with the path to this key.                    |
 | **preset-kyma-artifacts-bucket** | It defines the environment variable for the Kyma's artifact bucket.                     |
 | **preset-stability-checker-slack-notifications** | It defines a webhook URL and a client token required for the Slack integration.                 |
+| **preset-sap-slack-bot-token** | It sets the environment variable with the Slack token for the bot account in internal workspace. More information can bo found [here](https://api.slack.com/docs/token-types#bot).|
```
Comment (klaudiagrz) [inline docs/prow/presets.md]:
```suggestion
| **preset-sap-slack-bot-token** | It sets the environment variable for the Slack token of the bot account in the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot).|
```

File: development/tools/jobs/testinfra/prow_addons_ctrl_manager_test.go [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+	assert.True(t, actualPresubmit.Decorate)
+	assert.Equal(t, "github.com/kyma-project/kyma", actualPresubmit.PathAlias)
+	tester.AssertThatHasPresets(t, actualPresubmit.JobBase, tester.PresetDindEnabled, tester.PresetDockerPushRepoTestInfra, tester.PresetGcrPush, tester.PresetBuildPr)
+	assert.Equal(t, "^development/prow-addons-ctrl-manager/", actualPresubmit.RunIfChanged)
+	assert.Equal(t, tester.ImageGolangKubebuilderBuildpackLatest, actualPresubmit.Spec.Containers[0].Image)
+	assert.Equal(t, []string{"/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"}, actualPresubmit.Spec.Containers[0].Command)
+	assert.Equal(t, []string{"/home/prow/go/src/github.com/kyma-project/test-infra/development/prow-addons-ctrl-manager"}, actualPresubmit.Spec.Containers[0].Args)
+}
+
+func TestBucJobPostsubmit(t *testing.T) {
```
Comment (aszecowka) [inline development/tools/jobs/testinfra/prow_addons_ctrl_manager_test.go]: please sleep more

**Pull request merged**